common/metrics/defs.go

```diff
@@ -452,6 +452,10 @@ const (
 	MatchingAddActivityTaskScope
 	// MatchingAddDecisionTaskScope tracks AddDecisionTask API calls received by service
 	MatchingAddDecisionTaskScope
+	// MatchingAddActivityTaskScope tracks AddActivityTask API calls received by service
```

Comment:
It doesn't look right to use scope in this case, we may need new metrics not scopes